### PR TITLE
optimize code for show item with right layer

### PR DESCRIPTION
**Pull request by kyynk**
optimize code for show item with right layer

### CODE REVIEW

File: Source/Game/mygame_run.cpp [hunk 1]
```diff
@@ -278,20 +279,20 @@ namespace game_framework {
...
-		house1_2F_TR_chair._preX = house1_2F_TR_chair.GetPosX();
-		house1_2F_TR_chair._preY = house1_2F_TR_chair.GetPosY();
-		house1_2F_TL_chair.Reset();
-		house1_2F_TL_chair._preX = house1_2F_TL_chair.GetPosX();
-		house1_2F_TL_chair._preY = house1_2F_TL_chair.GetPosY();
-		house1_basement2_chair.Reset();
-		house1_basement2_chair._preX = house1_basement2_chair.GetPosX();
-		house1_basement2_chair._preY = house1_basement2_chair.GetPosY();
+		objs.at(obj_move::house1_2F_TR_chair).Reset();
+		objs.at(obj_move::house1_2F_TR_chair)._preX = objs.at(obj_move::house1_2F_TR_chair).GetPosX();
```
Comment (2021Vincent) [inline Source/Game/mygame_run.cpp:288]: all variable should be private, and should be assigned in class.

**Comment by kyynk** [comment]
optimize item id (define -> enum)

File: Source/Game/mygame_run.cpp (hunk 1 above)
Comment (2021Vincent) [reply Source/Game/mygame_run.cpp:288]: @kyynk

**Review by 2021Vincent — CHANGES_REQUESTED** [review]

**Comment by 2021Vincent** [comment]
oh, there is no need to @

**Review by 2021Vincent — APPROVED** [review]

**Pull request merged**